editor/questionbankcontroller.js

```diff
@@ -181,6 +193,37 @@ angular.module('createquestionapp', [])
       $scope.itemsLoading = true;
       $scope.searchQuestions();
       $scope.selectedIndex = undefined;
+      var qsManifest = org.ekstep.pluginframework.pluginManager.getPluginManifest($scope.pluginIdObj.question_set_id);
+      var qsVesrion = qsManifest.ver.split('.')[0];
+      var data = {
+        "request": {
+          "filters": {
+            "objectType": ["Content"],
+            "contentType": ["Plugin"],
+            "targets.id": $scope.pluginIdObj.question_set_id,
+            "targets.ver": {'<=': Number(qsVesrion)},
+            "status": "Live"
+          },
+          "limit": 50,
+          "fields": ['contentType','semanticVersion','appIcon']
+        }
+      };
+      ecEditor.getService('search').search(data, function(err, resp) {
+        var pluginsData = resp.data.result.content;
+        localStorage.setItem("qs-plugins", JSON.stringify(pluginsData));
+        var plugins = [];
+        ecEditor._.forEach(pluginsData, function(value, key) {
+          if (value) {
+            var obj = {
+              "id": value.identifier,
+              "ver": value.semanticVersion,
+              "type": 'plugin'
+            }
+            plugins.push(obj);
+          }
+        });
+        $scope.loadPlugins(plugins, []);
```

Comment:
Why are loading the plugin using scope level function? You can loadPlugin by calling ContentEditor API itself.